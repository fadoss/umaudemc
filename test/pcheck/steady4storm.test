# Steady-state probabilities specifically for Storm
#+ for: storm

# These are the test cases that do not converge with the default method of
# PRISM (see steady4prism), but they work out of the box with Storm

umaudemc pcheck coin head @steady --assign metadata
umaudemc pcheck coin head @steady --assign 'uaction(ttail=3, thead=7)'
umaudemc pcheck coin head @steady --assign @ite.assign
umaudemc pcheck coin head @steady 'choice(3 : ttail, 7 : thead) *' --assign strategy
