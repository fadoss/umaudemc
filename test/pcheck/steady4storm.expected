 0.615384615384615    head
 0.384615384615385    tail
 0.7                  head
 0.3                  tail
 0.7                  head
 0.3                  tail
Transient and steady-state probabilities are only computed for DTMCs.
